Trappist/src/Promact.Trappist.Core/Controllers/QuestionController.cs

```diff
@@ -50,6 +55,19 @@ public QuestionController(IQuestionRespository questionsRepository)
         {
             return Ok(await _questionsRepository.GetAllQuestionsAsync()); 
         }
+
+        /// <summary>
+        /// Returns all the coding languages
+        /// </summary>
+        /// <returns>
+        /// coding language object of type CodingLanguageAC
```

Comment:
Only first letter should be capital except for few cases. Comment is not proper.